### ISSUE TITLE
Replace golang.org links with go.dev links

### ISSUE THREAD

**Issue by Gauravpadam** [kubernetes/website#43926]
**This is a Bug Report**
Website uses [golang.org](https://www.golang.org) links which ultimately redirect to [go.dev](https://go.dev)
We do not know how long the redirects will be functional

It's better to replace these links with the go.dev domains

**Comment by AmarNathChary** [comment]
I see both Links you provided are redirecting to the same page [go.dev](https://go.dev/)

**Comment by AmarNathChary** [comment]
also, you can provide some of the page links for reference on the website.

**Comment by Gauravpadam** [comment]
Some references where we are still using golang.org
https://github.com/search?q=repo%3Akubernetes%2Fwebsite+path%3Acontent%2Fen+golang.org&type=code

**Comment by sftim** [comment]
I'm not sure this is a bug
/remove-kind bug
/kind cleanup

**Comment by AmarNathChary** [comment]
I have checked with some of the reference links its not working with go.dev

**Comment by AmarNathChary** [comment]
I replaced `golang.org` with `go.dev` it says it couldn't find the page.

**Comment by AmarNathChary** [comment]
example links https://pkg.go.dev/google.golang.org/grpc/balancer & https://pkg.go.dev/google.go.dev/grpc/balancer

**Comment by AmarNathChary** [comment]
The URLs cant be fixed by only changing `golang.org` with `go.dev` the whole URLs need to be updated some URLs may be unchanged.
example 
https://godoc.org/google.golang.org/grpc/balancer with this https://pkg.go.dev/google.golang.org/grpc/balancer
https://golang.org/pkg/unicode/#IsPrint with https://pkg.go.dev/unicode#IsPrint
https://tip.golang.org/doc/godebug with unchanged https://tip.golang.org/doc/godebug

**Comment by Gauravpadam** [comment]
That's correct @AmarNathChary, and that's the reason why I mentioned
> go.dev 'domains'

**Comment by Ritikaa96** [comment]
Hi @Gauravpadam Are you actively working on this issue?

**Comment by Gauravpadam** [comment]
No @Ritikaa96 I've just reported it

**Comment by Ritikaa96** [comment]
yes, I see!! @kumarankit999 are you actively working on it? or is it open for other contributors?
